docs/src/gauss_newton_kalman_inversion.md

```diff
@@ -0,0 +1,65 @@
+
+!!! note
+    GNKI relies on minimizing a loss function that includes regularization.  The user must specify a Gaussian prior with distribution ``\mathcal{N}(m, \Gamma_{\theta})``. See [Prior distributions](@ref parameter-distributions) to see how one can apply flexible constraints while maintaining Gaussian priors. 
+
+The optimal parameters ``\theta^*`` given relation (1) minimize the loss 
+
+ ```math
+\mathcal{L}(\theta, y) = \langle \mathcal{G}(\theta) - y \, , \, \Gamma_y^{-1} \left ( \mathcal{G}(\theta) - y \right ) \rangle + \langle m - \theta \, , \, \Gamma_{\theta}^{-1} \left ( m - \theta  \right ) \rangle,
+```
+
+where ``m`` is the prior mean and ``\Gamma_{\theta}`` is the prior covariance. 
+
+### Algorithm
+
+GNKI updates the ``j``-th ensemble member at the ``n``-th iteration by directly approximating the Jacobian with statistics from the ensemble.
+
+First, the ensemble covariance matrices are computed: 
+```math
+\begin{aligned}
+        &\mathcal{G}_n^{(j)}  = \mathcal{G}(\theta_n^{(j)}) \qquad 
+        \bar{\mathcal{G}}_n = \dfrac{1}{J}\sum_{k=1}^J\mathcal{G}_n^{(k)} \\
+        & C^{\theta \mathcal{G}}_n = \dfrac{1}{J - 1}\sum_{k=1}^{J}
+        (\theta_n^{(k)} - \bar{\theta}_n )(\mathcal{G}_n^{(k)} - \bar{\mathcal{G}}_n)^T \\
+        & C^{\theta \theta}_n = \dfrac{1}{J - 1} \sum_{k=1}^{J} 
+        (\theta_n^{(k)} - \bar{\theta}_n )(\theta_n^{(k)} - \bar{\theta}_n )^T.
+
+\end{aligned}
+```
+
+Using the ensemble covariance matrices, the update equation from ``n`` to ``n+1`` under GNKI is
+```math
+\begin{aligned}
+        & \theta_{n+1}^{(j)} = \theta_n^{(j)} + \alpha \left\{ K_n\left(y_n^{(j)} - \mathcal{G}(\theta_n^{(j)})\right) + \left(I - K_n G_n\right)\left(m_n^{(j)} - \theta_n^{(j)}\right) \right\} \\
+        
+        & \\
+
+        & K_n = \Gamma_{\theta} G_n^T \left(G_n \Gamma_{\theta} G_n^T + \Gamma_{y}\right)^{-1} \\
+        
+        & G_n = \left(C^{\theta \mathcal{G}}_n\right)^T \left(C^{\theta \theta}_n\right)^{-1}, 
+
+
+\end{aligned}
+```
+
+where ``y_n^{(j)} \sim \mathcal{N}(y, 2\alpha^{-1}\Gamma_y)`` and ``m_n^{(j)} \sim \mathcal{N}(m, 2\alpha^{-1}\Gamma_{\theta})``.
+
+## Creating the EKI Object
+
+We first build a prior distribution (for details of the construction see [here](@ref constrained-gaussian)). 
+Then we build our EKP object with `EnsembleKalmanProcess(args..., GaussNewtonInversion(prior); kwargs...)`.  For general EKP object creation requirements see [Creating the EKI object](@ref eki).  To make updates using the inversion algorithm see [Updating the Ensemble](@ref eki).  
```

Comment:
Could you use triple quotes: ` ` `  (without spaces) to get this to be a code block

Reply:
Formatting is all set!  I also included a couple minor changes that Matti suggested, so everything should be ready to merge!  Maybe you could review it one last time before I merge?